.changelog/2769.feature.2.md

```diff
@@ -0,0 +1,3 @@
+go/staking: add LastBlockFees query method
+
+LastBlockFees returns the collected fees for previous block.
```

Comment:
Slightly unrelated, but this is also something that should be part of the genesis state dump as otherwise it could cause the total token amount to become inconsistent on dump/restore.

Reply:
oh we don't do that at the moment :o will do a PR that adds that tomorrow :grin: